libs/clients/passports/src/lib/passports.config.ts

```diff
@@ -0,0 +1,28 @@
+import { defineConfig } from '@island.is/nest/config'
+import { NationalRegistryScope } from '@island.is/auth/scopes'
+import * as z from 'zod'
+
+const schema = z.object({
+  xRoadServicePath: z.string(),
+  fetch: z.object({
+    timeout: z.number().int(),
+    scope: z.array(z.string()),
+  }),
+})
+
+export const PassportsClientConfig = defineConfig<z.infer<typeof schema>>({
+  name: 'PassportsClient',
+  schema,
+  load(env) {
+    return {
+      xRoadServicePath: env.required(
+        'XROAD_PASSPORT_LICENSE_PATH',
+        'IS-DEV/GOV/10001/SKRA-Protected/Forskraning-V1',
+      ),
+      fetch: {
+        timeout: 10000,
+        scope: [NationalRegistryScope.individuals],
```

Comment:
Are they using the same scope on this passport service? 🤔

Reply:
Yes according to the information we got from Þjóðskrá it would be the same scope as for the individual service.
Do you feel like we should require a different scope for the passport service?

Reply:
Discussed offline that they should use the @skra.is/passport scope